CMakeLists.txt

```diff
@@ -283,13 +283,14 @@ if(ENABLE_TOOLS)
 endif()
 
 if(ENABLE_DATA_TOOLS)
+  pkg_check_modules(GEOS REQUIRED IMPORTED_TARGET geos)
   foreach(program ${valhalla_data_tools})
     get_source_path(path ${program})
     add_executable(${program} ${path})
     create_source_groups("Source Files" ${path})
     set_target_properties(${program} PROPERTIES FOLDER "Data Tools")
     target_include_directories(${program} PRIVATE ${cxxopts_include_dir})
-    target_link_libraries(${program} valhalla)
+    target_link_libraries(${program} valhalla PkgConfig::GEOS)
```

Comment:
With this change, the following piece of cmake should be removed - https://github.com/valhalla/valhalla/blob/32cba4fed382efa944e01f6f1eec8c92e3e38166/test/CMakeLists.txt#L29-L31

Reply:
And https://github.com/valhalla/valhalla/pull/5226#issuecomment-2847265397 is applicable here - linking valhalla to `GEOS` leads to introducing such dependency for router as well.

Reply:
> With this change, the following piece of cmake should be removed -

Ha, didn't see that, thanks!

> linking valhalla to GEOS leads to introducing such dependency for router as well.

We don't link `libvalhalla` to `geos`, only the data executables, which are most commonly run centrally and not distributed. But yeah, you're right it's still possible to distribute those and we should make a disclaimer already. Not sure if those license github apps can resolve those intricacies?

Reply:
> With this change, the following piece of cmake should be removed -

hm, wait, same argument: we're not linking `libvalhalla` to geos, so this should stay I think.